src/vs_config.cmd

```diff
@@ -0,0 +1,11 @@
+@echo off
+setlocal enabledelayedexpansion
+
+echo Configuring Visual Studio to build WiX Toolset...
+
+for /f "usebackq tokens=*" %%i in (`"C:\Program Files (x86)\Microsoft Visual Studio\Installer\vswhere.exe" -latest -property installationPath`) do (
+  echo Visual Studio installed at: "%%i"
+  rem "C:\Program Files (x86)\Microsoft Visual Studio\Installer\vs_installer.exe" modify --config "%~dp0wix.vsconfig" --installPath "%%i" --quiet --norestart --force
```

Comment:
Am I missing something? Starting the line with `rem` means the whole line is a comment so this batch file doesn't actually configure VS.

Reply:
;scream: